syz-manager/rpc.go

```diff
@@ -567,6 +576,27 @@ func (runner *Runner) sendSignalUpdate(plus, minus []uint32) error {
 	return flatrpc.Send(runner.conn, msg)
 }
 
+func (serv *RPCServer) startLeakChecking() {
+	serv.mu.Lock()
+	defer serv.mu.Unlock()
+	serv.checkLeaks = true
+	for _, runner := range serv.runners {
+		if runner.conn != nil {
+			runner.sendStartLeakChecks()
```

Comment:
Nit: it is theoretically possible that we could hang here for some time? E.g. if the target VM has hanged and it blocked the socket.

Reply:
Added a commit that does sends asynchronously.